src/utils/generateCommandTable.ts

```diff
@@ -0,0 +1,41 @@
+import extractString from "./extractString";
+import tableGenerator from "./tableGenerator";
+
+const HEADERS = ["Command", "Description"];
+const COMMAND_LINE_REGEX = /"\/\* .+ \*\/": ".+"/g;
+
+const START_OF_COMMAND = '"/* ';
+const END_OF_COMMAND = ' */": "';
+const END_OF_LINE = '"';
+const NO_DATA = "";
+
+/**  generates a table of CoCommands and Descriptions from a package.json
+ * @param {string} readmeContent - content of the package.json
+ * @returns {string} - Table of Commands
+ */
+
+export const generateCommandTable = (readmeContent: string): string => {
+  const data = new Array();
+  readmeContent.match(COMMAND_LINE_REGEX)?.forEach((line) => {
```

Comment:
What's the ? For

Reply:
@ralph-dev  there may not be any any command compatible to regex, so the object may become null